orchestration-shared/src/main/java/uk/gov/di/orchestration/shared/entity/VectorOfTrust.java

```diff
@@ -150,6 +150,14 @@ private static VectorOfTrust parseVtrSet(JSONArray vtrJsonArray) {
                                                                 "Invalid VTR attribute")));
     }
 
+    public static List<String> getRequestedLevelsOfConfidence(List<VectorOfTrust> vtrList) {
+        var levelsOfConfidence = new ArrayList<String>();
```

Comment:
This may be cleaner as a mapping, something like:
`return vtrList.stream().map(VectorOfTrust::getLevelOfConfidence).map(LevelOfConfidence::getValue).toList();`